src/utils/deprecationWarning.js

```diff
@@ -0,0 +1,8 @@
+export default function deprecationWarning(oldname, newname) {
+  if (process.env.NODE_ENV !== 'production') {
+    let message = `${oldname} is deprecated. Use ${newname} instead.`;
+    console.warn(message);
+    let link = 'https://github.com/react-bootstrap/react-bootstrap/issues/425#issuecomment-97110963';
```

Comment:
Since this message has use outside the Collapsable => Collapsible rename the link should probably be an argument that's passed in.